libs/lb/lb-rs/Cargo.toml

```diff
@@ -24,32 +24,34 @@ bezier-rs = "0.2.0"
 bincode = "1.3.3"
 time = "0.3.20"
 diffy = "0.3.0"
+db-rs = {git="https://github.com/lockbook/db-rs"}
+db-rs-derive = "0.3.1"
 indexmap = { version = "2.5.0", features = ["rayon"] }
 reqwest = { version = "0.11.1", default-features = false, features = ["json", "rustls-tls"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.44"
-uuid = { version = "1.2.2", features = ["v4", "serde"] }
+uuid = { version = "1.2.2", features = ["v4", "serde", "js"] }
 libsecp256k1 = "0.7.1"
 tracing = "0.1.5"
 tracing-subscriber = "0.3.9"
 tracing-appender = "0.2"
 sublime_fuzzy = "0.7.0"
 crossbeam = "0.8.1"
 qrcode-generator = "4.1.6"
-db-rs = "0.3.1"
-db-rs-derive = "0.3.1"
 strip-ansi-escapes = "0.2.0"
 chrono = "0.4"
 rand = "0.8.4"
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
does this remove `rand`? `rand` does support wasm iirc. Rand has cryptography implications iirc. Will continue to review.